freeipa/src/main/java/com/sequenceiq/freeipa/kerberosmgmt/v1/KerberosMgmtV1Service.java

```diff
@@ -64,4 +113,131 @@ public void deleteHost(HostRequest request) {
         //TODO  Will be implemented CDPSDX-515
         return;
     }
+
+    private FreeIpaClient getFreeIpaClient(Stack stack) {
+        String failure;
+        FreeIpaClient freeIpaClient = null;
+
+        try {
+            freeIpaClient = freeIpaClientFactory.getFreeIpaClientForStack(stack);
+        } catch (Exception e) {
```

Comment:
I don't see why you catch this exception here and not let it propagate. Also you turn it here into a general RuntimeException. This should be changed